### ISSUE TITLE
Add an application field or flag

### ISSUE THREAD

**Issue by HaroldHallikainen** [sandflow/melt#9]
I'd like a single database serving a bunch of industries. We would not use all the MESA tags in cinema, so we should be able to filter based on application.

**Comment by palemieux** [comment]
How would you describe the "application" for ISDCF?

**Comment by HaroldHallikainen** [comment]
I guess ISDCF would be good. People can allow records where this matches one or more options.


Thanks!


----
Harold Hallikainen
Engineer, QSC LLC
phone +1 720 360 4180
________________________________

**Comment by palemieux** [comment]
> I guess ISDCF would be good. People can allow records where this matches one or more options.

I am actually wondering what "ISDCF" means as an application. Does it mean "for use in identifying spoken and written languages in D-Cinema applications", or something else?

**Comment by HaroldHallikainen** [comment]
I think it would be limiting the number of languages to those used in cinema. The LMT list is huge. I think we should have one list for all media industries, but I don't think all those will be used in cinema.


Harold


----
Harold Hallikainen
Engineer, QSC LLC
phone +1 720 360 4180
________________________________

**Comment by palemieux** [comment]
To confirm, you are suggesting adding a label that identifies a tag as "for use in identifying spoken and written languages in D-Cinema applications", right?

**Comment by HaroldHallikainen** [comment]
Not really. We need the codes to identify written (subtitle or caption) and spoken (soundtrack). We have a few codes in the ISDCF list that are for one or the other, while most use the same code for both. but we should have a way to say "in cinema, use these codes" instead of presenting the user with a list of a thousand or more codes.


In all media, I think the language code represents the intended audience instead of being a precise description of the language. We may have something recorded in U.S. English (US dialect, spelling, etc.), but intended for use anywhere English is spoken. I think this should then be "en" instead of "en-US". If we make separate versions for different countries, then we could use "en-US", "en-GB", and "en" for everywhere else.


Thanks!


Harold


----
Harold Hallikainen
Engineer, QSC LLC
phone +1 720 360 4180
________________________________
